package.json

```diff
@@ -1,18 +1,24 @@
 {
   "scripts": {
     "test": "sass-lint static/**/*.scss --verbose --no-exit",
-    "build": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace static/css/styles.css && postcss static/css/styles.css --use cssnano > static/css/styles.min.css",
+    "build": "yarn run build-js && yarn run build-css",
+    "build-css": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace static/css/styles.css && postcss static/css/styles.css --use cssnano > static/css/styles.min.css",
+    "build-js": "cp node_modules/d3/build/d3.min.js static/js/modules && cp node_modules/topojson-client/dist/topojson-client.min.js static/js/modules",
     "watch": "watch -p 'static/sass/*.scss' -c 'yarn run build'",
```

Comment:
We should probably `watch` this to just `yarn run build-css`, as the JS doesn't need copying every time a Sass file changes.